### ISSUE TITLE
`yarn build` fails with error

### ISSUE THREAD

**Issue by j-funk** [metaplex-foundation/metaplex#904]
**Describe the bug**
`yarn build` command fails 

**To Reproduce**
1. run `yarn install && yarn bootstrap && yarn build`
2. see again by running `yarn build`

**Expected behavior**
I expect yarn to complete without errors.

**Terminal output**
```
> Build error occurred
/Users/jfunk/Documents/Code/git/metaplex/js/packages/web/next.config.js:41
    NEXT_ENABLE_NFT_PACKS: process.env.REACT_APP_ENABLE_NFT_PACKS,
    ^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Unexpected identifier
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at loadConfig (/Users/jfunk/Documents/Code/git/metaplex/js/node_modules/next/dist/next-server/server/config.js:8:72)
    at async Span.traceAsyncFn (/Users/jfunk/Documents/Code/git/metaplex/js/node_modules/next/dist/telemetry/trace/trace.js:6:584)
error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in 'web'
lerna WARN complete Waiting for 2 child processes to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


$ node --version
v16.7.0
```

**Desktop (please complete the following information):**
 - Branch: master
 - Device: Intel Macbook Pro, 2017
 - OS: OSX 11.6.1
 - Node 16.7.0
 - Metaplex 1.0.0

**Comment by J-a-m-e-s-o-n** [comment]
There is a comma missing at the end of line 40, will resolve the issue.

**Comment by J-a-m-e-s-o-n** [comment]
https://github.com/metaplex-foundation/metaplex/pull/903

**Comment by j-funk** [comment]
@J-a-m-e-s-o-n thank you, that error is solved, however, now we are failing with a different error related to typescript:

Looks like this command: `tsc -p ./src` is causing the failure

```
lerna info Executing command in 4 packages: "yarn run build"
lerna info run Ran npm script 'build' in '@oyster/common' in 22.9s:
$ run-s build-ts build-css
$ tsc
$ less-watch-compiler src/ dist/lib/ --run-once
Running less-watch-compiler once.
lerna ERR! yarn run build exited 2 in '@metaplex/cli'
lerna ERR! yarn run build stdout:
$ tsc -p ./src
node_modules/typescript/lib/lib.dom.d.ts(25,1): error TS6200: Definitions of the following identifiers conflict with those in another file: NodeFilter, XPathNSResolver, SVGMatrix, WebKitCSSMatrix, SVGPoint, SVGRect, location, webkitURL, ImportExportKind, TableKind, ValueType, ExportValue, Exports, ImportValue, Imports, ModuleImports, ElementTagNameMap, name, AlgorithmIdentifier, BigInteger, BlobPart, BodyInit, BufferSource, COSEAlgorithmIdentifier, CanvasImageSource, ConstrainBoolean, ConstrainDOMString, ConstrainDouble, ConstrainULong, DOMHighResTimeStamp, DOMTimeStamp, EventListenerOrEventListenerObject, Float32List, FormDataEntryValue, GLbitfield, GLboolean, GLclampf, GLenum, GLfloat, GLint, GLint64, GLintptr, GLsizei, GLsizeiptr, GLuint, GLuint64, HTMLOrSVGImageElement, HTMLOrSVGScriptElement, HashAlgorithmIdentifier, HeadersInit, IDBValidKey, ImageBitmapSource, InsertPosition, Int32List, LineAndPositionSetting, MediaProvider, MessageEventSource, MutationRecordType, NamedCurve, OnBeforeUnloadEventHandler, OnErrorEventHandler, PerformanceEntryList, ReadableStreamController, ReadableStreamDefaultReadResult, ReadableStreamReader, RenderingContext, RequestInfo, TexImageSource, TimerHandler, Transferable, Uint32List, UvmEntries, UvmEntry, VibratePattern, WindowProxy, AlignSetting, AnimationPlayState, AppendMode, AttestationConveyancePreference, AudioContextLatencyCategory, AudioContextState, AuthenticatorAttachment, AuthenticatorTransport, AutoKeyword, AutomationRate, BinaryType, BiquadFilterType, CanPlayTypeResult, CanvasDirection, CanvasFillRule, CanvasLineCap, CanvasLineJoin, CanvasTextAlign, CanvasTextBaseline, ChannelCountMode, ChannelInterpretation, ClientTypes, ColorSpaceConversion, CompositeOperation, CompositeOperationOrAuto, CredentialMediationRequirement, DOMParserSupportedType, DirectionSetting, DisplayCaptureSurfaceType, DistanceModelType, DocumentReadyState, EndOfStreamError, EndingType, FillMode, FullscreenNavigationUI, GamepadHapticActuatorType, GamepadMappingType, IDBCursorDirection, IDBRequestReadyState, IDBTransactionMode, ImageOrientation, ImageSmoothingQuality, IterationCompositeOperation, KeyFormat, KeyType, KeyUsage, LineAlignSetting, MediaDeviceKind, MediaKeyMessageType, MediaKeySessionType, MediaKeyStatus, MediaKeysRequirement, MediaStreamTrackState, NavigationType, NotificationDirection, NotificationPermission, OrientationLockType, OrientationType, OscillatorType, OverSampleType, PanningModelType, PaymentComplete, PermissionName, PermissionState, PlaybackDirection, PositionAlignSetting, PremultiplyAlpha, PublicKeyCredentialType, PushEncryptionKeyName, PushPermissionState, RTCBundlePolicy, RTCDataChannelState, RTCDtlsTransportState, RTCIceCandidateType, RTCIceComponent, RTCIceConnectionState, RTCIceCredentialType, RTCIceGathererState, RTCIceGatheringState, RTCIceProtocol, RTCIceTcpCandidateType, RTCIceTransportPolicy, RTCIceTransportState, RTCPeerConnectionState, RTCRtcpMuxPolicy, RTCRtpTransceiverDirection, RTCSdpType, RTCSignalingState, RTCStatsIceCandidatePairState, RTCStatsType, ReadyState, ReferrerPolicy, RequestCache, RequestCredentials, RequestDestination, RequestMode, RequestRedirect, ResidentKeyRequirement, ResizeObserverBoxOptions, ResizeQuality, ResponseType, ScrollBehavior, ScrollLogicalPosition, ScrollRestoration, ScrollSetting, SelectionMode, ServiceWorkerState, ServiceWorkerUpdateViaCache, ShadowRootMode, SpeechSynthesisErrorCode, TextTrackKind, TextTrackMode, TouchType, UserVerificationRequirement, VideoFacingModeEnum, VisibilityState, WebGLPowerPreference, WorkerType, XMLHttpRequestResponseType```

**Comment by j-funk** [comment]
Had a version of TypeScript installed globally that was interfering with my build.

**Comment by AnhNPT** [comment]
Hello, i had this error similar to you?
How do you fix that?

**Comment by AnhNPT** [comment]
> The 1st error was fixed by using the version from #903 - I fixed the 2nd error by removing the global installation of TypeScript

Thanks for that
